hosting_docs/source/installation/1-quick-monolith-install.rst

```diff
@@ -34,7 +34,17 @@ SSH into the server with a root or a user with root privileges and do below.
 	# fill the config and save
 	vim install-config.yml
 
-3. Run the installation script. (You may be prompted for sudo password for some of the commands)
+3. Optional: setup with Python 3.10 instead of 3.6
```

Comment:
The idea of quick-install is that we provide least number of steps and hence provide sensible defaults. As such, quick-install should reflect the latest stable setup that we use on our environments. So, may be we don't need to provide this option here?

Reply:
However if we do want to give the user the option, the ideal way to do this without adding the extra step is 
- To provide this as a config option in `config.yml`
- Use that to set the python version in the [install script](https://github.com/dimagi/commcare-cloud/blob/master/quick_monolith_install/cchq-install.sh) and pass that to any other commands inside it such as init.sh.

Reply:
Ah meant to push a commit that would have changed this to default to 3.10, though to your point that still would have cluttered the quick-install steps. Realistically, there shouldn't be any reason to provide an option for 3.6. I removed this step entirely in 949b779d1604332009ba927bd68b1802cd6df16a, and plan to test the monolith install with 3.10 before these docs go live.

Reply:
Great, thank you!